core/primitives/src/test_utils.rs

```diff
@@ -681,4 +681,10 @@ impl FinalExecutionOutcomeView {
             );
         }
     }
+
+    /// Calculates how much NEAR was burnt for gas, after refunds.
+    pub fn gas_cost(&self) -> Balance {
```

Comment:
The name suggests the returned value would be gas, rather than NEAR. This wouldn't as big of a deal if Balance was a newtype, but with both it and Gas being plain aliases to regular integers (even if of different bitwidths) this can easily slip through.

Lets find at least a better name for this method.